app/api/__init__.py

```diff
@@ -10,6 +11,13 @@
 # users
 api.route(UserList, 'user_list', '/users')
 api.route(UserDetail, 'user_detail', '/users/<int:id>')
```

Comment:
Isn't that `/computers/<int:id>/relationships/owner`?
This is here: https://github.com/fossasia/open-event-orga-server/pull/3704/files#diff-f4c05149e2001f8002755304b9c14c80R19
and
https://github.com/fossasia/open-event-orga-server/pull/3704/files#diff-f4c05149e2001f8002755304b9c14c80R31